src/main/java/org/rumbledb/expressions/quantifiers/QuantifiedExpression.java

```diff
@@ -80,10 +80,18 @@ public QuantifiedOperators getOperator() {
 
     @Override
     public String serializationString(boolean prefix) {
-        return "";
+        StringBuilder sb = new StringBuilder();
+        sb.append(quantifier.equals(Quantification.EVERY) ? "every " : "some ");
+        String separator = "";
+        for (QuantifiedExpressionVar v : this.variables) {
+            sb.append(separator);
+            sb.append(v.toString());
+            separator = ", ";
+        }
+        return sb.toString();
     }
 
-    public enum QuantifiedOperators {
+    public enum Quantification {
```

Comment:
This is absolutely a non-issue. I just wanted to bring something I noticed to your attention. The name 'quantifier' is chosen for variables (for example in the constructor) while the enum is called Quantification. How do you feel about the name 'Quantifier' for the enum?